phoenix-core/src/main/java/org/apache/phoenix/mapreduce/util/IndexColumnNames.java

```diff
@@ -61,6 +68,9 @@ public IndexColumnNames(final PTable pdataTable, final PTable pindexTable) {
         if (pindexTable.getViewIndexId() != null) {
             offset++;
         }
+        if (forTenant && pindexTable.isMultiTenant()) {
```

Comment:
@gokceni - Would the forTenant make a difference here? What if it were a multi-tenant global view index? The _IDX table would still be <ViewIndexId, TenantId, Other indexed columns>, but presumably forTenant would be false.

Reply:
Sure. I did not want to break it for the existing code. That is why I added forTenant. Will remove

Reply:
Should probably also have a test for a global view index on a multi-tenant table to make sure that combination works, if we don't already have one.